modules/exploits/windows/smb/ms17_010_eternalblue.rb

```diff
@@ -187,6 +187,10 @@ def smb_eternalblue(process_name, grooms)
       print_status("Receiving response from exploit packet")
       code, raw = smb1_get_response(sock)
 
+      if code.nil?
+        print_error("Did not receive a response from exploit packet")
```

Comment:
should this be a `fail_with`? not sure what the state of things looks like if we failed to get a response. can we succeed from this point if that's the case?